bower.json

```diff
@@ -28,7 +28,7 @@
     "selectize": "0.12.1",
     "c3": "0.4.11",
     "kubernetes-label-selector": "1.4.4",
-    "kubernetes-container-terminal": "1.0.4",
+    "kubernetes-container-terminal": "git@github.com:jwforres/container-terminal.git#xterm",
```

Comment:
Needs to switch to the release once the upstream PR https://github.com/kubernetes-ui/container-terminal/pull/25 is merged